### ISSUE TITLE
GPU problem

### ISSUE THREAD

**Issue by Friday131** [junqiangchen/LiTS---Liver-Tumor-Segmentation-Challenge#25]
My GPU is RTX 2070   8G,   when I run vnet3d_train with patch(256,256,16), it fails, error log liike that:
Caused by op 'output/Conv3D', defined at:
  File "D:/py/LiTS---Liver-Tumor-Segmentation-Challenge-master/LiTS/vnet3d_train.py", line 42, in <module>
    train()
  File "D:/py/LiTS---Liver-Tumor-Segmentation-Challenge-master/LiTS/vnet3d_train.py", line 36, in train
    Vnet3d = Vnet3dModule(256, 256, 16, channels=1, costname=("dice coefficient",))
  File "D:\py\LiTS---Liver-Tumor-Segmentation-Challenge-master\LiTS\Vnet\model_vnet3d.py", line 203, in __init__
    self.phase, self.drop)
  File "D:\py\LiTS---Liver-Tumor-Segmentation-Challenge-master\LiTS\Vnet\model_vnet3d.py", line 152, in _create_conv_net
    output_map = conv_sigmod(x=layer9, kernal=(1, 1, 1, 32, n_class), scope='output')
  File "D:\py\LiTS---Liver-Tumor-Segmentation-Challenge-master\LiTS\Vnet\model_vnet3d.py", line 52, in conv_sigmod
    conv = conv3d(x, W) + B
  File "D:\py\LiTS---Liver-Tumor-Segmentation-Challenge-master\LiTS\Vnet\layer.py", line 51, in conv3d
    conv_3d = tf.nn.conv3d(x, W, strides=[1, stride, stride, stride, 1], padding='SAME')
  File "C:\Users\aiage\Miniconda3\envs\vnet_tensor\lib\site-packages\tensorflow\python\ops\gen_nn_ops.py", line 1487, in conv3d
    name=name)
  File "C:\Users\aiage\Miniconda3\envs\vnet_tensor\lib\site-packages\tensorflow\python\framework\op_def_library.py", line 787, in _apply_op_helper
    op_def=op_def)
  File "C:\Users\aiage\Miniconda3\envs\vnet_tensor\lib\site-packages\tensorflow\python\framework\ops.py", line 3392, in create_op
    op_def=op_def)
  File "C:\Users\aiage\Miniconda3\envs\vnet_tensor\lib\site-packages\tensorflow\python\framework\ops.py", line 1718, in __init__
    self._traceback = self._graph._extract_stack()  # pylint: disable=protected-access

InternalError (see above for traceback): Blas SGEMM launch failed : m=1048576, n=1, k=32
	 [[Node: output/Conv3D = Conv3D[T=DT_FLOAT, data_format="NDHWC", dilations=[1, 1, 1, 1, 1], padding="SAME", strides=[1, 1, 1, 1, 1], _device="/job:localhost/replica:0/task:0/device:GPU:0"](add_8, outputW/read/_139)]]
	 [[Node: Neg/_143 = _Recv[client_terminated=false, recv_device="/job:localhost/replica:0/task:0/device:CPU:0", send_device="/job:localhost/replica:0/task:0/device:GPU:0", send_device_incarnation=1, tensor_name="edge_2696_Neg", tensor_type=DT_FLOAT, _device="/job:localhost/replica:0/task:0/device:CPU:0"]()]]

And what can I do with it ?

**Comment by Friday131** [comment]
I have solve it, it's the incompatibility problem, CUDA9.0 must put some patchs
